.github/workflows/submit.yml

```diff
@@ -55,6 +55,7 @@ jobs:
         with:
           path: .output/*.zip
           if-no-files-found: error
+          include-hidden-files: true
```

Comment:
Ah yeah, haven't used this since this action started ignoring hidden files. Thanks!